.github/CODEOWNERS

```diff
@@ -61,5 +56,5 @@
 /tests/testdata/modules/images/dominant-color                               @pbearne @spacedmonkey
 
 # Module: Full Page Cache Health Check
-/modules/site-health/audit-full-page-cache                                  @manuelRod @westonruter
-/tests/modules/site-health/audit-full-page-cache                            @manuelRod @westonruter
+/modules/object-cache/audit-full-page-cache                                  @manuelRod @westonruter
+/tests/modules/object-cache/audit-full-page-cache                            @manuelRod @westonruter
```

Comment:
Nit-pick, but please keep the indentation of the owner usernames here aligned between the different modules, similar to how it was before.

Reply:
Thanks for the review. indentation corrected.